plugins/GigPlayer/GigPlayer.cpp

```diff
@@ -531,7 +539,11 @@ void GigInstrument::loadSample( GigSample& sample, sampleFrame* sampleData, f_cn
 	}
 
 	unsigned long allocationsize = samples * sample.sample->FrameSize;
+#ifndef _MSC_VER
 	int8_t buffer[allocationsize];
+#else
```

Comment:
You can also add `_freea(buffer)` within the ifdef i posted in the other review comment.